example/src/main/resources/application.properties.example

```diff
@@ -30,6 +30,7 @@ spring.datasource.initialization-mode=always
 
 # The SQL dialect makes Hibernate generate better SQL for the chosen database
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL94Dialect
+spring.jpa.properties.hibernate.globally_quoted_identifiers=true
```

Comment:
i would say this isn't needed, the things that this would affect are things we already define, the side effects that comes with it aren't really worth it

Reply:
The limitation this works around is in DatabaseDefinitionService.generateBranchSchemaFromModels().  Here, hibernate DDL is creating the Nodes* tables before the select into gets to run.  Hibernate can't create a table with a dash without this set.

Reply:
ah, I remember this now - I also commented on the jira - there's a SuffixNamingStrategyClass that we can set this for just table names instead of doing it for everything

Reply:
actually would be better if it's just set for nodes* tables and just leave other tables as is